tests/test_pipdeptree.py

```diff
@@ -465,6 +465,8 @@ def test_parser_svg():
 
 @pytest.mark.parametrize("args_joined", [True, False])
 def test_custom_interpreter(tmp_path, monkeypatch, capfd, args_joined):
+    import virtualenv
```

Comment:
This should be at the top of the file

Reply:
I moved it down to get the AssertionError from setuptools that was reported in #162.
I moved it up again but now the test passes even without the changes in the module from this PR.

Reply:
Hmm, then we'll need to change it around, but this test sounds very brittle :thinking: perhaps rework it into a subprocess call to not make it depend on setuptools being imported?

Reply:
Yeah it is a weird one.
I gave it a try with subprocess now, how does that look?